js/everything.js

```diff
@@ -84,6 +85,33 @@ function onPlayerReady(event) {
     // Setup player
     if (currentVideoID) {
         ZenPlayer.init(currentVideoID);
+        setupPlyr();
+    }
+}
+
+function setupPlyr() {
+    //set up Plyr player
+    plyrPlayer = plyr.setup({
+        autoplay: true,
+        controls:["play", "current-time", "duration", "mute", "volume", "captions"]
+    })[0];
+    //Load video into Plyr player
+    if (plyrPlayer) {
+        plyrPlayer.source({
+            type: "video",
+            title: player.getVideoData().title,
+            sources: [{
+                src: currentVideoID,
+                type: "youtube"
+            }]
+        });
+        //Hide video; leave only controls
+        $(".plyr__video-wrapper").hide();
+        //Inject svg with controls' icons
```

Comment:
There are and there will be issues with this while running the site locally (in `file` protocol). JS won't ever allow to access `file` protocol (websites would be able to steal everyone's data). `.load()` uses the same `XMLHttpRequest` but it's just one line instead of four.
I use python's `http.server` to quickly and easily host the site locally. Of course Youtube API has some issues when doing so.